keras_cv/utils/preprocessing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+def transform_to_standard_range(images, value_range):
```

Comment:
I think this interface is somehow confusing, since the input param `value _range` is not the standard_range, and the default standard range [0, 255] is hidden/implied in the implementation.

My original thought was to have it like, where user need to explicitly specify the from/to value range.

```
def convert_image_plx_value(images, from_range, to_range)
```